tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
Defining a copy operation provides a way to produce two objects that are equal under any reasonable definition, but it doesn't tell you anything about the equality of objects that don't have equivalent members.  For example, take `Time` (which should be in for review later today).  It has copy semantics and an equality operator, but the value of the equality operator is in no way guessable from the copy operator.  Or consider again `EmbeddingMap`, where it is non-obvious whether `Identity` and `AffineMap(0,1,0,1)` should be equal (or even `AffineMap(0,1,0,1)` and `AffineMap(0,2,0,2)`).

> Now, one may argue that we should never need to compare instances of a specific class, in which case my response would be "Why should we be able to copy it then?"

I just answered that: Because we may need to call two functions that both take it by value.

You're going to have to explain to me why these clauses are related, since you seem to find it obvious and I don't.  For example, if you replaced the word "compare" with "dereference" I assume you would no longer agree with it, but the two sentences make equal sense to me.